<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heydemanndata</class>
 <widget class="QDialog" name="heydemanndata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCALabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoName</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IK-Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Analog Values</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0 (M)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>1 (M)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>2 (M)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>3 (M)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>4 (G)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>5 (G)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>6 (G)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>7 (G)</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Correction Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Write to file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0-3</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set to Zero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>4-7</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Grating</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Grating</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0-7</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Both</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Both</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>431</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculation</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:aoHeydSignal.PROC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>390</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
